barretenberg/cpp/src/barretenberg/crypto/pedersen_hash/c_bind_new.cpp

```diff
@@ -5,13 +5,8 @@
 
 extern "C" {
 
-WASM_EXPORT void pedersen_hash(uint8_t const* inputs_buffer, uint8_t* output)
-{
-    std::vector<grumpkin::fq> to_compress;
-    read(inputs_buffer, to_compress);
-    auto r = crypto::pedersen_hash::hash(to_compress);
-    barretenberg::fr::serialize_to_buffer(r, output);
-}
+WASM_EXPORT void pedersen_hash_init() {}
+
```

Comment:
I just moved this to be above `pedersen_hash_with_hash_index` because thats how it is ordered in the header file